packages/components/src/components/contract-card/contract-card-items/contract-update-form.tsx

```diff
@@ -15,8 +14,24 @@ import Icon from '../../icon';
 import MobileWrapper from '../../mobile-wrapper';
 import Money from '../../money';
 import InputWithCheckbox from '../../input-wth-checkbox';
+import { TGetCardLables, TToastConfig } from '../../types';
+import { TContractStore } from '@deriv/shared/src/utils/contract/contract-types';
+
+export type TContractUpdateFormProps = {
+    addToast: (toast_config: TToastConfig) => void;
+    contract: TContractStore;
+    current_focus?: string;
+    error_message_alignment: string;
+    getCardLabels: TGetCardLables;
+    onMouseLeave: () => void;
+    removeToast: (toast_id: string) => void;
+    setCurrentFocus: (name: string) => void;
+    status: string;
+    toggleDialog: (e: any) => void; // This function accomodates events for various HTML elements, which have no overlap, so typing it to any
```

Comment:
@jim-deriv Could we use `unknown`? 🤔 The TS would force us to do a type check before using it.

Reply:
@farzin-deriv I already tried 😄, didn't work

Reply:
@jim-deriv Then let’s move on 🙇🏻 Thanks man 🌹